react/components/DeliveryOptionsAvailability.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent, Fragment } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { Alert } from 'vtex.styleguide'
+
+interface Props {
+  numberOfItems: number
+  numberOfUnavailableItems: number
+}
+
+const DeliveryOptionsAvailability: FunctionComponent<Props> = ({
+  numberOfItems,
+  numberOfUnavailableItems,
+}) => {
+  const allItemsUnavailable = numberOfUnavailableItems === numberOfItems
+  return (
+    <Fragment>
+      {numberOfUnavailableItems ? (
```

Comment:
In general we use the `&&` without returning `null` in this scenarios.

Reply:
ditto